src/ResourceManager/IotCentral/Commands.IotCentral/Microsoft.Azure.Commands.IotCentral.generated.format.ps1xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0"?>
+<Configuration>
+  <ViewDefinitions>
+    <View>
+      <Name>Microsoft.Azure.Commands.IotCentral.Models.PSIotCentralApp</Name>
+      <ViewSelectedBy>
+        <TypeName>Microsoft.Azure.Commands.IotCentral.Models.PSIotCentralApp</TypeName>
+      </ViewSelectedBy>
+      <TableControl>
+        <TableHeaders>
+          <TableColumnHeader>
+            <Alignment>Left</Alignment>
+            <Label>Resource Name</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Alignment>Left</Alignment>
+            <Label>Subdomain</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Alignment>Left</Alignment>
+            <Label>Display Name</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Alignment>Left</Alignment>
+            <Label>Location</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Alignment>Left</Alignment>
+            <Label>Template</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Alignment>Left</Alignment>
+            <Label>Tags</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Alignment>Left</Alignment>
+            <Label>Sku Name</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Alignment>Left</Alignment>
+            <Label>Resource Group Name</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Alignment>Left</Alignment>
+            <Label>Application Id</Label>
+          </TableColumnHeader>
+        </TableHeaders>
+        <TableRowEntries>
+          <TableRowEntry>
+            <TableColumnItems>
+              <TableColumnItem>
+                <Alignment>Left</Alignment>
```

Comment:
Can you really see all of these properties on the regular PowerShell screen? This seems like a lot of properties for table format.

Reply:
@maddieclayton they all fit for me, ill check again

Reply:
@maddieclayton you're right it doesn't all show on a small powershell, but it autoformats to only show the columns that will fit. is it better for me to reduce the amount of data output all together?

Reply:
@cle5eland As long as the columns that are shown are the ones that you want, that's fine with me.

Reply:
Cool, thanks